KabakiAntony/Politico
# #165592502  finishes the user sign up page

#### What does this PR do?
It merges the finished sign up page with gh-pages

#### Description of Task to be completed?
The page has been made mobile first and the other screen sizes adopted with growth.

#### How should this be manually tested?
Just go to https://KabakiAntony.github.io/Politico/UI/user-signup.html on any of the devices that have access to the internet and see how it renders on them differently.

#### What are the relevant pivotal tracker stories?
#165592502

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/49396576/57701553-ec9c8a80-7664-11e9-9cb5-6c33f680b6a5.png)
